packages/react-native-editor/src/test/api-fetch-setup.test.js

```diff
@@ -50,6 +54,31 @@ describe( 'isPathSupported', () => {
 			} );
 		} );
 	} );
+
+	describe( 'POST requests', () => {
+		unsupportedPaths.POST.forEach( ( path ) => {
+			it( `does not support ${ path }`, () => {
+				expect( isPathSupported( path, 'POST' ) ).toBe( false );
+			} );
+		} );
```

Comment:
This test case doesn't check supported paths as we don't have any for POST requests yet.